paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -686,6 +688,7 @@ def get_authors(self) -> str:
             )
         # If there's no production deploy group, or the production deploy group
         # has never been deployed to, just use the old SHA from this deploy group.
+        # TODO: do we need to get author information from image_version-only bumps?
```

Comment:
this comment was also removed from one of the previous image_version commits, and can be removed here.

Reply:
Removed.